### ISSUE TITLE
 ownershipNode derived from wrong source (?)

### ISSUE THREAD

**Issue by gavinpatkinson** [urbit/keygen-js#5]
According to previous implementation and design:  derive a wallet from the ownerSeed directly

**Comment by gavinpatkinson** [comment]
Ok, I looked into this. @Fang- 

Currently: 
```js
  const ownershipNode = await childNodeFromSeed({
    seed: ownerSeed,
    type: 'owner',
    revision: null,
    ship: null,
    password: password,
  });
```

Resolves to:
```js
{
    meta: {
      type,
      revision: ...
      ship: ...l
    },
    seed: ...,
    keys: await walletFromSeed(childSeedBuffer, password),
  };
```
Where `childSeedBuffer` is `ownerSeed`
Before: 

```js
let ownerPromise = walletFromSeed(ownerSeed, password);
```
which is later resolved: 
```js
result.owner.keys  = await ownerPromise;
```

I think the bigger questions is: Do we want metadata fields in the ownership node?

**Comment by Fang-** [comment]
Github is not showing a comment from you which I did receive an email for, so just in case you didn't intentionally delete that and want an answer:

The "before" case doesn't include metadata because there really is no metadata to speak of here. The ownership seed wasn't derived using a salt, so everything would be null. Both the "null meta" and "no meta" cases are inconsistent with other seeds, but afaic that's fine.

**Comment by gavinpatkinson** [comment]
I realized that whole post was wrong, and I resolved the issue locally. childNodeFromSeed() uses childSeedFromSeed() which is not how the ownership node was initially derived,.

**Comment by gavinpatkinson** [comment]
Resolved in #7
